RationaleMCP/0029/ReadMe.md

```diff
@@ -0,0 +1,79 @@
+
+Currently, the license information is neither considered at all (even if available as documentation or resource) for exported models or based on some analysis of tool vendors to create at least some kind of exported license information.
+This proposal improves the situation both for tool vendors as well as library developers to create legally correct exported models.
+A library developer needs to provide the relevant license information as external resource, which on the other hand can be considered by a Modelica tool during the export process.
+
+Use-cases:
+1. Export a Modelica model with Modelica.Blocks.Continuous.PID as FMU for ME and distribute this FMU to external customers or partners.
+Along with the FMU the distributor needs to provide the information of utilized software components, e.g., the MSL, which as of v4.0.0 is licensed under the 3-Clause BSD license.
+1. Export a Modelica model with Modelica.Blocks.Sources.CombiTable1D as S-Function and distribute this S-Function to external customers or partners.
+Along with the S-Function the distributor needs to provide the information of utilized third-party software of the ModelicaStandardTables, zlib, uthash etc.
+1. Export a Modelica model with Modelica.Blocks.Continuous.PID as FMU for CS and distribute this FMU to external customers or partners.
+Along with the FMU the distributor needs to provide the information of utilized software components, e.g., the utilized solver.
+The license conditions of the solver component need to be respected, e.g., in case of the SUNDIALS CVode solver the relevant license information should be made obvious for the FMU receiver by means of an appropriate license file/text.
+
+Remark: Compared to the first two aspects, the third aspect solely is a tool issue, but stated here for completeness.
+
+## Backwards Compatibility
+
+This change does not introduce any backwards incompatibilities.
+
+## Tool Implementation
+
+### Experience with Prototype
+
+On the library side, the necessary changes have been applied to library _ExternData_.
+The following commits at the Git repository at https://github.com/modelica-3rdparty/ExternData demonstrate the necessary steps to update a Modelica libarry.
+
+1. https://github.com/modelica-3rdparty/ExternData/commit/7522046dce6d20fe01a85a559b02fd9d8878bdd2
+   - All license files were added as text files to a new directory ExternData/Resources/Licenses.
+     The file format is not restricted to Text with a special encoding, but any file format (e.g., Mark-Down, PDF, HTML) basically can be supported.
+     The license file LICENSE_ExternData.txt is the corresponding license of the copyright holder of the Modelica package, all other license files reflect third-party software components.
+   - It is recommended to use ModelicaPackage/Resource/Licenses as default directory for the license files.
```

Comment:
What does _default_ mean here, is this a recommendation to users to put their files here or is the tool expected to do anything special with this _default_ directory?

Reply:
Either use this directory or the one specified by the License annotation.

Reply:
It's still no very clear what you mean, but my interpretation is that you should remove the word _default_ from that sentence.

Reply:
Agreed. The recommended makes it clear that it can be overriden.

Reply:
File has been updated.